pkg/xds/topology/outbound.go

```diff
@@ -54,10 +54,15 @@ func BuildEndpointMap(
 			tlsEnabled = externalService.Spec.Networking.Tls.Enabled
 		}
 
+		tags := externalService.Spec.GetTags()
+		if tlsEnabled {
+			tags[mesh_proto.ServiceTag+"_tls"] = "enabled"
```

Comment:
I'd advise this:
```
tags[`kuma.io/external-service-name`] = externalService.GetName()
```
in a second you will introduce more TLS settings so simple enabled/disabled is not enough

Reply:
Well, should the tag denote that there is TLS involved? This sounds like if tlsEnabled is true, we just set the tag with the external service name.

Reply:
It's not even visible in the UI or anywhere.

The whole idea is that if you have
```
type: ExternalService
mesh: default
name: httpbin-1
tags:
  kuma.io/service: httpbin
  version: "1"
networking:
  address: httpbin.org:80
  tls:
    enabled: true
    caCert:
      secret: sec-1
```
and
```
type: ExternalService
mesh: default
name: httpbin-2
tags:
  kuma.io/service: httpbin
  version: "2"
networking:
  address: httpbin.org:80
  tls:
    enabled: true
    caCert:
      secret: sec-2
```

you've got different TLS settings for different ExternalService. Simple `_tls` is not enough to differentiate both of them.

You can set the
```
tags[`kuma.io/external-service-name`] = externalService.GetName()
```
regardless if TLS is on or off.